frontend/src/App.tsx

```diff
@@ -115,40 +153,41 @@ const App: React.FC = () => {
                   }
                 }}
               ></video>
-              Hooray! You're connected!
+              Media path - media connected...
             </div>
           </div>
         ) : (
           <div>
-            <span>connecting...</span>
+            <span>Media path - awaiting connection...</span>
           </div>
         )}
         <div>
-          <div
-            style={{
-              display: "flex",
-              justifyContent: "center",
-              alignItems: "center",
-            }}
-          >
-            <span>click to call from {voiceApplicationPhoneNumber}</span>
-            <button
-              style={{ height: "30px", marginLeft: "10px" }}
-              disabled={outboundPhoneNumber?.length === 0}
-              onClick={callOutboundPhoneNumber}
-            >
-              CALL
-            </button>
-            <input
-              type="text"
-              name="numberToDial"
-              id="numberToDial"
-              placeholder="enter a phone number"
-              style={{ height: "30px", marginLeft: "10px" }}
-              onChange={updateTn}
-            />
+          <div>
+            <span>Action:</span>
+            {callState === "idle" ? (
+              <React.Fragment>
+                <button
+                  style={{ height: "30px", marginLeft: "10px" }}
+                  disabled={outboundPhoneNumber?.length === 0}
+                  onClick={callOutboundPhoneNumber}
+                >
+                  CALL
+                </button>
+                <input
+                  type="text"
+                  name="numberToDial"
+                  id="numberToDial"
+                  placeholder="enter a phone number"
+                  style={{ height: "30px", marginLeft: "10px" }}
+                  onChange={updateTn}
+                />
+              </React.Fragment>
+            ) : (
+              <span>. . . . . .</span>
```

Comment:
The Action:...... looked a little weird in the browser. I don't have a specific suggestion but I would recommend something different

Reply:
Agreed - I'll have to mess with it.

Reply:
cleaned up a bit - no more 'Action' if there is no action that is possible.  Still room for explicit answer and hangup if we get around to it, but fearing that it is already too complex.